src/components/shared/Navigation.tsx

```diff
@@ -372,7 +372,7 @@ const WalletTab = createNavigatorForTab(
     },
     [RouteName.ReferralBonusPage]: {
       screen: ReferralBonus,
-      navigationOptions: createHeaderNavigationOptions()
+      navigationOptions: createHeaderNavigationOptions("Mint referral bonus", true)
```

Comment:
Can we just call it Referrals? Seems like a long header that could get chopped off on smaller phones

Reply:
done